server/konnectors/malakoff_mederic.coffee

```diff
@@ -0,0 +1,174 @@
+        genOptions =
+            url: engineUrl
+            jar: true
+
+        # Request a js page to extract a generated session id: because why not
+        request genOptions, (err, res, body) ->
+            if err
+                return next 'request error'
+
+            regexp = /dwr.engine._origScriptSessionId = "([A-Z0-9]+)"/g
+            matches = body.match regexp
+            id = matches[0].split('"')[1]
+            # The client must generate 3 random digits
+            scriptSessionId = id + Math.floor(Math.random() * 1000)
+
+            path = "/dwr/call/plaincall/InternauteValidator.checkConnexion.dwr"
+            submitUrl = "#{domain}#{path}"
+            checkOption =
+                method: 'POST'
+                jar: true
+                url: submitUrl
+                headers:
+                    'Content-Type': 'text/plain'
+                body:
+                    "callCount=1\n\
+                    page=/espaceClient/LogonAccess.do\n\
+                    httpSessionId=#{httpSessionId}\n\
+                    scriptSessionId=#{scriptSessionId}\n\
+                    c0-scriptName=InternauteValidator\n\
+                    c0-methodName=checkConnexion\n\
+                    c0-id=0\n\
+                    c0-param0=boolean:false\n\
+                    c0-param1=string:#{requiredFields.login}\n\
+                    c0-param2=string:#{requiredFields.password}\n\
+                    batchId=0\n"
+
+            # Log in
+            request checkOption, (err, res, body) ->
+                if err
+                    log.error err
+                    return next 'request error'
+                else if res.statusCode >= 400
+                    log.error 'Authentication error'
+                    return next 'request error'
+                # The body should not contain LOGON_KO
+                else if body.indexOf('LOGON_KO') > -1
+                    log.error 'Authentication error'
+                    return next 'bad credentials'
+
+                log.info 'Logged in'
+
+                path = "/espaceClient/sante/tbs/redirectionAction.do"
+                reimbursementUrl = "#{domain}#{path}"
+                options =
+                    method: 'GET'
+                    url: reimbursementUrl
+                    jar: true
+
+                # Get the reimbursements
+                request options, (err, res, body) ->
+                    if err
+                        logger.error err
+                        return next 'request error'
+
+                    data.html = body
+                    next()
+
+
+# Parse the fetched page to extract bill data.
+parsePage = (requiredFields, healthBills, data, next) ->
+
+    healthBills.fetched = []
+    return next() if not data.html?
+
+    $ = cheerio.load data.html
+
+    $('.headerRemboursements').each ->
+        amount = $(this).find('.montant').text()
+        amount = amount.replace(' €', '')
```

Comment:
Can you also replace the ',' by '.' so that what's after the coma is not ignored by the parseFloat amount line, and the link with the bank operations is correctly done.